### ISSUE TITLE
[BUG] New Journal presents a blank screen

### ISSUE THREAD

**Issue by rogercreagh** [TechbeeAT/jtxBoard#613]
**Describe the bug**
This is a strange one and has been occurring at least since v2.4.2 (possibly 2.4.1) and is still  present in 2.4.3. It is only affecting one device which is running Android 11 and has an eInk screen so jtxB is set to use e-ink theme. The other device is Android 12 with a colour screen. The DavX5 setup on both is the same (v4.3.3 which arrived on fDroid today)

When starting a new journal entry from the list view the screen is initially blank. It remains blank for a variable time, never less than 30 secs, sometimes more than 2 mins and then refreshes itself and the edit page appears.

You can make it appear by clicking the cogwheel bottom left to bring up settings - the screen appears partially hidden and greyed out by the settings pane.When you dismiss the settings pane all is normal.

I said that the Davx5 setup is identical, but there might be something I've missed. Most likely perhaps it is something to do with failing to paint the e-ink screen before something else (?initial sync poll?) happens since there is never a problem on the non-eInk device.

Apologies for not reporting this sooner but I have been away more or less since v2.4.2 was released. During much of that time I had sync turned off since I had no network connection, and it was still the same so perhaps not a DavX5 interaction problem.

I see a similar sounding issue has been raised over the widget sometimes not painting the screen. (#116) 

**Device and version:**
 - Device: Boox NovaAir Tablet
 - Android 11]
 - jtx Board version:  2.4.2 and 2.4.3
 - Downloaded from:  f-Droid

**Comment by patrickunterwegs** [comment]
Thanks, I'll have a look. It is more likely related to some data handling of the UI framework and that the data fetching die not trigger a refresh of the UI anymore (until you do another action as you describe).

**Comment by rogercreagh** [comment]
Any progress, it is not specifically linked to the e-ink theme as it does not happen if I select e-ink theme on a devicewith a normal screen. But it does only happen on a device with an e-inkscreen.

It was definitely not there before v2.4.0 but arrived with 2.4.2 (I can't remember if it was there in 2.4.1)

It is getting seriously annoying and close to making jtxB hard to use on e-ink screens. It always happens when creating a new item, and sometimes when simply viewing an existing item from the list.

**Comment by patrickunterwegs** [comment]
Hi, I will have a look at it soon, basically I can only assume that a library version upgrade causes a different behavior than before...

**Comment by patrickunterwegs** [comment]
Another user reported the same issue on a Samsung Galaxy S21 fe, Android 13, Samsung Galaxy Tab S7 fe, Android 13

@rogercreagh regrettably when reviewing the code I couldn't find anything problematic. And regrettbly I can't reproduce the issue reliably. I you can find any constellation that let's you reproduce the problem persistently, I'd be grateful if you could let me know.

As you say it appears ALWAYS on the tablet, I'll do another check specifically with Android 11 (although I also saw the issue on Android 13, but then it was not reproducible...)

**Comment by patrickunterwegs** [comment]
Alright, after hours of experimenting I think I found a solution. It's really pretty random when this issue occurs. 
I hope this change will resolve it for good (4b32f7ec1887aa543a3032527665316e23704aeb)

**Comment by rogercreagh** [comment]
Well on Saturday I updated both devices to Davx5 4.3.4.1 and jtxB 2.04.07

Since then the tablet device where it was happening consistently now is clear - it hasn't recurred (but this is the main live device so I've only created 3 journal entries and edited a couple and viewed a few since updating - previously though it would have happened every time when creating and occasionally when viewing.

However on the phone where I also create and delete test entries it has happened at least once since updating - but previously I had never seen it on that device. There may have been different other apps open though.
